torch/csrc/api/include/torch/all.h

```diff
@@ -21,3 +21,4 @@
 #include <torch/types.h>
 #include <torch/utils.h>
 #include <torch/version.h>
+#include <torch/xpu.h>
```

Comment:
If users do not build XPU(USE_XPU=0), what's the behavior?

Reply:
It is Ok if USE_XPU=0 since the function in xpu.h is implemented via hooks.